scripts/build-docs.sh

```diff
@@ -1,46 +1,97 @@
 #!/bin/bash
 
 # Path of the directory with the documentation
-SOURCE_DIR="website/site/"
+SOURCE_DIR="site/"
 S3_CMD_CFG_LOCATION=${S3_CMD_CFG_LOCATION:-"$HOME/.s3cfg"}
 S3_DOCS_BUCKET_NAME=${S3_DOCS_BUCKET_NAME:-"docs.tf.k2.cloud"}
 
+help () {
+    echo "
+    --help  Show this message and exit.
+    --tools Install all required dependencies.
+    --push  Build and push internal documentation to the bucket.
+    --local Build and run internal documentation locally."
+}
+
+tools () {
+  python3 -m venv .venv
+  source .venv/bin/activate
+  pip3 install mkdocs mkdocs-material mkdocs-awesome-pages-plugin | pip install mkdocs mkdocs-material mkdocs-awesome-pages-plugin
+  deactivate
+}
+
 # Copying of configuration for the documentatiion to the website folder
-cp -r docs/c2/mkdocs/assets docs/c2/mkdocs/images website/docs/
-cp docs/c2/mkdocs/mkdocs.yml website/
+copy () {
+  cp -r mkdocs/images docs/mkdocs_images
+  cp -r mkdocs/assets docs/mkdocs_assets
```

Comment:
```suggestion
  cp -r mkdocs/images docs/mkdocs-images
  cp -r mkdocs/assets docs/mkdocs-assets
```

Рядом просто `data-sources/` живет )